lib/main_common.pm

```diff
@@ -625,24 +625,27 @@ sub load_jeos_tests {
         loadtest "jeos/prepare_firstboot";
     }
 
-    load_boot_tests();
+    load_boot_tests() unless get_var('NO_CLOUD');
     if (check_var('FIRST_BOOT_CONFIG', 'combustion')) {
         loadtest 'microos/verify_setup';
         loadtest 'microos/image_checks';
     } elsif (check_var('FIRST_BOOT_CONFIG', 'cloud-init')) {
-        loadtest "boot/cloud_init";
+        loadtest "installation/first_boot";
+        loadtest 'jeos/verify_cloudinit';
     } else {
         loadtest "jeos/firstrun";
+        if (get_var('POSTGRES_IP')) {
+            loadtest "jeos/image_info";
+        }
+
     }
-    if (get_var('POSTGRES_IP')) {
-        loadtest "jeos/image_info";
-    }
+
     loadtest "jeos/record_machine_id";
     loadtest "console/force_scheduled_tasks";
     # this test case also disables grub timeout
     loadtest "jeos/grub2_gfxmode";
     unless (get_var('INSTALL_LTP') || get_var('SYSTEMD_TESTSUITE')) {
-        loadtest "jeos/diskusage" unless is_openstack;
+        loadtest "jeos/diskusage" unless get_var('FLAVOR') =~ /OpenStack-Cloud/;
```

Comment:
Why is this limited to this flavor?

Reply:
because of `xfs`, that is the only image with it

Reply:
Ah yes right. Mind adding a comment?

Reply:
Added